py/serve.py

```diff
@@ -768,6 +768,10 @@ def init_user():
                                   port=int(POSTGRES_PORT))
     if flask.request.path in ('/login', '/authenticate'):
         return
+    if 'user' in flask.request.args:
```

Comment:
This is good, but can we also check for existing old-style cookies and issue access tokens for those? I'm thinking of the case where a user navigates to `editor.open-reaction-database.org` without using a URL that has a GET parameter.

Reply:
Done.

Reply:
nit: have the URL take precedence?